sig-storage/annual-report-2021.md

```diff
@@ -0,0 +1,164 @@
+  * Non-graceful node shutdown: https://github.com/kubernetes/enhancements/pull/1116
+  * Allow Kubernetes to supply pod's fsgroup to CSI driver on mount: https://github.com/kubernetes/enhancements/pull/2323
+  * Volume group: https://github.com/kubernetes/enhancements/pull/1551
+  * Recover from resize failures: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1790-recover-resize-failure
+  * SELinux recursive permission handling: https://github.com/kubernetes/enhancements/issues/1710
+  * Volume/Snapshot namespace transfer: https://github.com/kubernetes/enhancements/pull/2326
+  * Volume health (2nd alpha due to re-design): https://github.com/kubernetes/enhancements/pull/2286
+  * Prioritization on volume capacity (sig-scheduling): https://github.com/kubernetes/enhancements/pull/1862
+  * PVC created by statefulset will not be auto removed (sig-apps): https://github.com/kubernetes/enhancements/pull/1915
+  * Volume expansion for Stateful sets (sig-apps): https://github.com/kubernetes/enhancements/pull/1848
+  * Enable User namespaces in kubelet so UIDs get shifted (sig-node): https://github.com/kubernetes/enhancements/pull/2101
+ContainerNotifier (sig-node): https://github.com/kubernetes/enhancements/pull/1995
+  * User id ownership in configmaps and secrets (sig-auth)
+
+What areas and/or subprojects does the group need the most help with?
+* There are features that we co-owned with sig-node, sig-apps, etc. We would need help from other SIGs to move things forward.
+  * Features co-owned with sig-node:
+    * ContainerNotifier: https://github.com/kubernetes/enhancements/pull/1995
+    * Volume health: https://github.com/kubernetes/enhancements/pull/2286
+  * Features co-owned with sig-apps:
+    * PVC created by statefulset will not be auto removed: https://github.com/kubernetes/enhancements/pull/1915
+    * Volume expansion for statefulset: https://github.com/kubernetes/enhancements/pull/1848
+  * Features co-owned with sig-scheduling:
+    * Prioritization on volume capacity (sig-scheduling): https://github.com/kubernetes/enhancements/pull/1862
+
+* There are also areas where we need sig-architecture’s help. For example, VolumeSnapshot is already GA and it is a core feature in sig-storage developed using CRDs. We have documented and blogged that components such as the snapshot CRDs and snapshot controller need to be deployed by Kubernetes distros, however, not every distro is installing them. This is a problem with any feature developed using CRD. We hope sig-architecture can help us resolve this problem.
+
+What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?
+
+According to the following stats on 1/25/2021, it shows the 7 day MA for a PR Time to Approve and Merge in sig-storage repository group is:
+Open to lgtm: avg 3.55 days, max 14.11 week
```

Comment:
nit: might be good to add this as bullet points because it isn't formatted well in the rendered markdown

Reply:
Fixed.